tsl/test/expected/bgw_scheduler_restart.out

```diff
@@ -67,7 +67,7 @@ SELECT pg_reload_conf();
 SHOW timescaledb.bgw_scheduler_restart_time;
  timescaledb.bgw_scheduler_restart_time 
 ----------------------------------------
- 1min
+ 10s
```

Comment:
Hmmm I think to make sure it will never be flaky we should reconnect after reload the configuration. It  is flaky probably because signaling all sessions about the new configuration can take some time o and then we're reading the old GUC value. Does it make sense for you?